tools/SchemaManager.Core/SqlSchemaManager.cs

```diff
@@ -142,7 +142,12 @@ await Policy.Handle<SchemaManagerException>()
 
                     string script = await GetScriptAsync(executingVersion, availableVersion.ScriptUri, token, availableVersion.DiffUri).ConfigureAwait(false);
 
-                    await UpgradeSchemaAsync(executingVersion, script, token).ConfigureAwait(false);
+                    // check if the record for given version exists in failed status
```

Comment:
It looks like we use the diff scripts when availableVersions doesn't start with "1". I think the FHIR service doesn't include 1 and 2 anymore, does that mean we'll always initialize it using the diff scripts?